packages/Liquid-Core.package/LQPasswordManager.class/instance/initialize.st

```diff
@@ -0,0 +1,3 @@
+initialize-release
+initialize
+	store := Dictionary new.
```

Comment:
[sand] Maybe we can find a better name for `store`, perhaps something not as general? For me, `passwordStore` would do it already